package.json

```diff
@@ -6,10 +6,7 @@
   "license": "MIT",
   "author": "IBM Corp.",
   "copyright.owner": "IBM Corp.",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/loopbackio/loopback-next.git"
-  },
+  "repository": "github:loopbackio/loopback-next",
```

Comment:
We're using the "git" syntax across the other packages, which is needed for the "directory" field for this monorepo. Is there a benefit of using the short syntax?

Reply:
It's just something more visual. For the rest you have to continue using the long one.

https://docs.npmjs.com/cli/v8/configuring-npm/package-json#repository